### ISSUE TITLE
Add isolation variables for all sensor layers to SvtTrack

### ISSUE THREAD

**Issue by omar-moreno** [omar-moreno/hps-dst#17]
As noted by @meeg in his email: 

> TrackData now has 12 doubles which are the isolations in all sensor layers. Same sign convention and special values (to indicate "no other hit in this layer" or "no hit in this layer") as before.

Currently, SvtTrack only has isolation variables for layers 1 and 2 so it needs to be modified so it can encapsulate the isolation variables for all layers. This will also require a modification to SvtDataWriter so all variables are written to the SvtTrack.

**Comment by omar-moreno** [comment]
An array of doubles of length 12 was added to SvtTrack in order to store the isolation variables.  SvtDataWriter was modified to check that the TrackData data structure contains 12 doubles (the isolation variables) and writes them to SvtTrack.  

The change currently lives in the branch hps-dst_Issue17.  They will be merged into the master once the pass 2 reconstruction of the data is done.

This issue is resolved as of commit a41a7cc1d39bc4515b8aaaaf8ffe1cd3061453d5 .
